.circleci/split_suites.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+ls -1 apps/*/test/*SUITE.erl test/*SUITE.erl > suites.txt
+cat suites.txt | sed 's/.*\///g' | sed 's/.erl//g' > classes.txt
+circleci tests split --split-by=timings --timings-type=classname classes.txt > batch.txt
+grep -f batch.txt suites.txt | paste -s -d ',' -
```

Comment:
Why? It's temp files used to produce the suites split.

I would do it without all those files, but didn't spent too much time.

Reply:
The point is that as Developer I want to be confident that CI is considering all jobs hence have CI to list all tests so that I can check that. And having intermediate steps would enable debugging it.

Though totally optional and debatable so ok.

Reply:
Most of this can be run and debugged locally (you can install `circleci` CLI). The only gotcha is that you have to extract the timing data from previous builds manually.